com.googleplaymusicdesktopplayer.GPMDP.json

```diff
@@ -7,8 +7,6 @@
   "runtime-version": "18.08",
   "sdk": "org.freedesktop.Sdk",
   "command": "google-play-music-desktop-player",
-  "rename-desktop-file": "google-play-music-desktop-player.desktop",
-  "rename-icon": "google-play-music-desktop-player",
```

Comment:
I'm very confused why this passed the buildbot but the application is just missing this icon surely?

Reply:
It has an icon on my machine, I started it via Gnome-Shell's overview and it had the expected icon.

Reply:
I don't know why it passed the buildbot, but it was the only way to get it to pass (the buildbot results are not exactly verbose with their error messages).

I had different approaches to copying the image the the right folder but none worked, all ended with the build failing because it couldn't find the icon.

The desktop file the application comes with references just "google-play-music-desktop-player" as its icon and that is the name the icon has in the package so maybe it does find it? I'm not 100% sure what exactly "rename-icon" / "rename-desktop-file" do, maybe that did introduce some error?

I'll have another look at it but I am afraid it will fail the build again.